tests/integrations/tests/utils/parametrize.py

```
@@ -0,0 +1,65 @@
+
+This module provides pytest parametrization for testing across multiple AI providers
+with automatic scenario-based filtering.
+"""
+
+from typing import List, Tuple
+from .config_loader import get_config
+
+
+def get_cross_provider_params_for_scenario(
+    scenario: str,
+    include_providers: List[str] = None,
```

Ruff (0.14.3)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)